doc/nrf/config_and_build/bootloaders_and_dfu/fw_update.rst

```diff
@@ -329,18 +351,63 @@ If this image has, in order of precedence, a *major*, *minor*, or *revision* val
 .. note::
 
    The optional label or build number specified after the ``+`` character is ignored when evaluating the version.
-   An existing application image with version ``0.1.2+3`` can be overwritten by an uploaded image with ``0.1.2+2``, but not by one with ``0.1.1+2``.
+   For example, an existing application image with version ``0.1.2+3`` can be overwritten by an uploaded image with ``0.1.2+2``, but not by one with ``0.1.1+2``.
```

Comment:
Also I think build number(what  comes after the + ) doesn't count when you do this comparison see: https://github.com/nrfconnect/sdk-mcuboot/blob/adab597a0eb0eb9c030a7b797748a49ca89988c2/boot/bootutil/src/loader.c#L685 
You need to enable a special option for that.

Don't know if this is important to mention

Reply:
So if I enable CONFIG_MCUBOOT_VERSION_CMP_USE_BUILD_NUMBER, ``0.1.2+2`` won't overwrite ``0.1.2+3``?

Reply:
> So if I enable CONFIG_MCUBOOT_VERSION_CMP_USE_BUILD_NUMBER, `0.1.2+2` won't overwrite `0.1.2+3`?

IIRC yes. We may mention it in doc.